data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
+				accept
+	on complete
+		conversation
+			`	You leave the ship with Anna following you. The moment you are out of your ship, three Kimek surround you and Anna. The Kimek in the middle speaks, and then brief silence. The silence is broken as their translation box speaks.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
+			`	You and Anna introduce yourself. Anna's introduction is longer, while you only need to state your name, seeing as you are more familiar with the Coalition now.`
+			`	"About what, do you write?", asks the Kimek.`
+			`	"Oh, I write science fiction books," says Anna.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
+			`	Then, switching back to your language, the Kimek explains that Anna is welcome to be with you, but states that she will not be allowed to do much unless she registers.`
+			`	"Register?", asks Anna.`
+			`	"You have to register yourself with the Coalition before you can do most stuff here. I'll take you to the Ring of Friendship, that's where you can complete the registration process."`
+			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`
+
+mission "Fangvin: Struggling Author 2"
+	name "Help Anna Register"
+	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	to offer
+		has "Fangvin: Struggling Author 1: done"
+	on offer
+		conversation
+			`You walk through the spaceport, looking for Anna. She is at a restaurant, but has ordered nothing with a disgusted look on her face. "These aliens! They vomit on their food and slurp all the soup! Disgusting!", she mutters.`
+			choice
+				`	"That's just how they eat."`
+				`	"My ship still has a decent amount of food you can eat."`
+				`	"There should be another restaurant."`
+			`	"<first>! It's you again!", says Anna in an excited tone. "Is the ship ready? Remember, we need to go to the <destination> so I can get registered with the Heliarchs."`
+			choice
+				`	"It's all ready now. Come."`
+			`	Anna stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, to peer into space. "What a massive ringworld!", she exclaims.`
+			choice
+				`	"It's enormous. They actually stole it from the Quarg, who they rebelled against roughly six thousand years ago."`
```

Comment:
They rebelled 7000 years ago, 6000 years ago was when they finally drove out the last of the Quarg.